nuget/framework/nunit.nuspec

```diff
@@ -30,7 +30,6 @@ Supported platforms:
     <dependencies>
       <group targetFramework="netstandard1.6">
         <dependency id="NETStandard.Library" version="1.6.0" />
-        <dependency id="Microsoft.NETCore.Portable.Compatibility" version="1.0.1" />
```

Comment:
@ChrisMaddock I was targeting just what was required for Xamarin, but you are right, so I removed the reference from the remaining projects and from the NUnitLite nuspec. The reference is still needed in two of our test projects.